build_rocm_python3

```diff
@@ -21,7 +21,7 @@ TF_PKG_LOC=/tmp/tensorflow_pkg
 rm -f $TF_PKG_LOC/tensorflow*.whl
 
 # First positional argument (if any) specifies the ROCM_INSTALL_DIR
-ROCM_INSTALL_DIR=/opt/rocm-5.1.1
+ROCM_INSTALL_DIR=/opt/rocm-5.1.0
```

Comment:
If you want to change the ROCM version you have to also change it in the container so the build job doesn't fail:

https://github.com/ROCm/tensorflow-upstream/blob/r1.15-denorm/tensorflow/tools/ci_build/Dockerfile.rocm#L6
https://github.com/ROCm/tensorflow-upstream/blob/r1.15-denorm/tensorflow/tools/ci_build/Dockerfile.rocm#L9